Api 4872 properties changelog (#524)
livechat/livechat-public-docs: content/messaging/agent-chat-api/changelog/index.mdx

### Changed

> - Method `update_customer` now has no response payload instead of Customer object

Reviewer: ```suggestion
- Method `update_customer`, which used to return the Customer object, now has no response payload.
```

> - Method `get_chat_threads` now requires `thread_ids` and no longer returns threads summary in response

Reviewer: ```suggestion
- Method `get_chat_threads` now requires `thread_ids`. Also, it no longer returns `threads_summary` in the response.
```

> - `properties` object is now the same in request and in response/push

Reviewer: ```suggestion
- The `properties` data structure now has the same format in requests as in responses and pushes.
```
